docs/01-app/01-getting-started/02-layouts-and-pages.mdx

```diff
@@ -86,10 +85,6 @@ export default function DashboardLayout({ children }) {
 
 The layout above is called a [root layout](/docs/app/api-reference/file-conventions/layout#root-layouts) because it's defined at the root of the `app` directory.
 
-### Root Layout (Required)
-
-The root layout is defined at the top level and wraps all routes. This layout is **required** and must contain `html` and `body` tags.
```

Comment:
Should we mention somewhere that the root layout is required? Now that this is removedi dont see it mentioned unless im missing it?

Reply:
Good point, will add that it's required. I removed it because this section was a duplicate of the one already in the API reference.